sru/release-25/dist-upgrade-test.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+
+cat >install-24.3-uat.yaml <<EOF
+#cloud-config
+ssh_import_id: [lamoura]
+apt:                                                                            
+  sources:                                                                      
+    testingpro:
+      source: "ppa:chad.smith/test-ua-client-do-release-upgrade"
```

Comment:
We should be able to just use the daily ppa instead of my personal test ppa once #1172  merges. So let's wait on that landing to address this.